File: SpringBootApp/src/main/java/com/example/springbootapp/data/dto/OrderDto.java
package com.example.springbootapp.data.dto;

import com.example.springbootapp.data.entities.Embdeddables.OrderInformations;
import com.example.springbootapp.data.entities.Enums.OrderStatus;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;
import java.util.List;

@Data
@NoArgsConstructor
public class OrderDto {
    private String id;
    private Double totalPrice;
    private LocalDateTime date;
    private OrderStatus status;
    private OrderInformations resilientInfos;
    private List<OrderItemDto> items;
}

File: ClientAdmin/app/src/main/java/com/example/clientadmin/model/dto/OrderDto.kt
package com.example.clientadmin.model.dto

import com.example.clientadmin.model.embdeddables.OrderInformations
import com.example.clientadmin.model.enumerator.OrderStatus
import java.time.LocalDate

data class OrderDto (
    val id: String,
    val totalPrice: Double,
    val date: LocalDate,
    val status: OrderStatus,
    val resilientInfos: OrderInformations,
    val items: List<OrderItemDto>
)